### PR TITLE
test: add unit test for SendButton component

### PR DESCRIPTION
**Pull request by yaminmomo15**
#### Description

Added unit test for SendButton component.

Closes #([142](https://github.com/tjtanjin/react-chatbotify/issues/142))

#### What change does this PR introduce?

Please select the relevant option(s).

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (changes to docs/code comments)
- [x] Unit Test

#### What is the proposed approach?

Implement unit test using Jest and react testing library.

#### Checklist:

- [x] The commit message follows our adopted [guidelines](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Testing has been done for the change(s) added (for bug fixes/features)
- [x] Relevant comments/docs have been added/updated (for bug fixes/features)

### CODE REVIEW

File: __tests__/components/buttons/SendButton.test.tsx [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+		expect(button).toHaveStyle(`backgroundColor: ${DefaultSettings.general?.secondaryColor}`);
+
+		// Simulate mouse leave
+		fireEvent.mouseLeave(button);
+
+		// Check initial color after mouse leave
+		expect(button).toHaveStyle(`backgroundColor: ${DefaultSettings.general?.primaryColor}`);
+	});
+
+	it("renders default aria-label and default style color while testArea is enabled", () => {
```
Comment (tjtanjin) [inline __tests__/components/buttons/SendButton.test.tsx:80]: Typo here, `textArea`

**Review by tjtanjin — CHANGES_REQUESTED** [review]
LGTM, just a small nit to fix 😝

File: __tests__/components/buttons/SendButton.test.tsx (hunk 1 above)
Comment (yaminmomo15) [reply __tests__/components/buttons/SendButton.test.tsx]: I just fixed it.

**Review by tjtanjin — APPROVED** [review]
LGTM! 😊

**Pull request merged**